src/main/java/com/github/caldav4j/model/request/CalendarMultiget.java

```diff
@@ -133,9 +130,7 @@ protected Namespace getNamespace() {
                 || calendarDataProp != null) {
 			Prop temp = new Prop();
 			temp.addChildren(properties.getChildren());
-            if (calendarDataProp != null){
```

Comment:
Why remove this check?

Reply:
There's no need to remove the check, I'll revert this change right away.